### PR TITLE
SC-4713: Introduced the URL to get OpenAPI schema for all Glue endpoints.

### PR DESCRIPTION
**Pull request by sprymiker**
### Description

- Ticket/Issue: https://spryker.atlassian.net/browse/SC-4713
- RG: https://release.spryker.com/release-groups/view/2961

#### Change log

- Introduced `/schema.yml` URL for all Glue endpoints.
- Adjusted Swagger UI to take a schema from the introduced URL.

### Checklist
- [X] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: bin/standalone/cli-rpc.js [hunk 1]
```diff
@@ -5,10 +5,95 @@ const http = require('http');
...
+
+    _options_glueSchema(request, response) {
+        response.setHeader('Access-Control-Allow-Origin', '*');
+        response.setHeader('Access-Control-Allow-Methods', 'GET');
+        response.statusCode = 201;
+        response.end();
+    }
+
+    _get_glueSchema(request, response) {
+        const fileLocation = process.env.SPRYKER_REST_API_SCHEMA_PATH || 'src/Generated/Glue/Specification/spryker_rest_api.schema.yml';
```
Comment (alexanderM91) [inline bin/standalone/cli-rpc.js:64]: What will be if the file doesn't exist

Comment (alexanderM91) [reply bin/standalone/cli-rpc.js:64]: Also, why it's not handled by the application, as a separate endpoint?

File: generator/src/templates/nginx/http/gateway/glue.server.conf.twig [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{% extends "nginx/http/gateway/server.conf.twig" %}
+{% block locations %}
+{{ parent() }}
+    location /schema.yml {
```
Comment (alexanderM91) [inline generator/src/templates/nginx/http/gateway/glue.server.conf.twig:4]: Also, why it's not handled by the application, as a separate endpoint?

File: bin/standalone/cli-rpc.js (hunk 1 above)
Comment (sprymiker) [reply bin/standalone/cli-rpc.js:64]: It will be 404.

File: generator/src/templates/nginx/http/gateway/glue.server.conf.twig (hunk 2 above)
Comment (sprymiker) [reply generator/src/templates/nginx/http/gateway/glue.server.conf.twig:4]: This solution is ready and does not need any efforts.

**Review by alexanderM91 — APPROVED** [review]

**Review by zyuzka — APPROVED** [review]

**Pull request merged**